manifests/globals.pp

```diff
@@ -30,6 +30,21 @@
   $pidfilepath           = undef,
   $pidfilemode           = undef,
   $manage_pidfile        = undef,
+
+  $opsmanager_user           = undef,
```

Comment:
I'm leaning to leaving all these out. I want to do the same for other classes

Reply:
Do you mean the user and group undef declarations or all of the undef declarations?

Reply:
I meant all `$opsmanager_*` variables from `globals.pp`.